src/Idris/Main.idr

```diff
@@ -1,35 +1,27 @@
 module Main
 
-import Core.Binary
-import Core.Context
+import Data.Strings
+import System
```

Comment:
Is there a convention on ordering imports in Idris ? I feel like the version before this change followed non-strict alphabetical order so i just wonder if it is convention or personal taste ?

Reply:
I wait, didn't mean to commit this, will ammend.

Reply:
haha, didn't realised this was mistake. But good work in general 👍